.github/workflows/tests.yaml

```diff
@@ -1,18 +1,89 @@
 name: Tests
 on: [push, pull_request]
 jobs:
-  test:
+  test-linux:
     strategy:
+      fail-fast: false
       matrix:
-        os: [ubuntu-latest, windows-latest]
-    runs-on: ${{ matrix.os }}
+        target:
+        - linux-amd64-unit-test-4-cpu
```

Comment:
It would be better we have the following three targets to cover the no-race cases:
- linux-amd64-unit-test-1-cpu
- linux-amd64-unit-test-2-cpu
- linux-amd64-unit-test-4-cpu

Of course, it can be addressed in a separate PR if there is no any objection.

Reply:
Will do!